components/ActivityCardPreview/index.js

```diff
@@ -0,0 +1,46 @@
+import Image from "next/image";
+import styled from "styled-components";
+import getFilteredCategories from "@/utils/filterCategories";
+
+export const StyledCateogriesUl = styled.ul`
+  list-style: none;
+  padding: 5px;
+  display: flex;
+  justify-content: center;
+  gap: 10px;
+`;
+
+export const StyledCategory = styled.li`
+  border: 1px solid grey;
+  padding: 5px;
+  border-radius: 5px;
+  margin: 10px 0px;
+`;
+
+const StyledImage = styled(Image)`
+  width: auto;
+  height: auto;
+`;
+
+export default function ActivityCardPreview({ activity }) {
+  const { title, categoryIds, imageUrl } = activity;
+  const filterCategories = getFilteredCategories(categoryIds);
+  return (
+    <>
+      <h2>{title}</h2>
```

Comment:
Don't skip h1 on this page
```suggestion
      <h1>{title}</h1>
```

Reply:
we already have an h1 in the parent element, so when rendering there would be two h1s.